experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -46,27 +46,33 @@ end
 """
    land_albedo(slab_sim::BucketSimulation)
 
-Returns the surface albedo of the earth; 
-a method for the bucket model 
+Returns the surface albedo of the earth;
+a method for the bucket model
 when used as the land model.
 """
 function land_albedo(slab_sim::BucketSimulation)
-    α_land =
-        surface_albedo.(
-            slab_sim.integrator.p.bucket.α_sfc,
-            slab_sim.params.albedo.α_snow,
-            slab_sim.integrator.u.bucket.σS,
-            slab_sim.params.σS_c,
-        )
+    α_land = ClimaLSM.surface_albedo(slab_sim.model, slab_sim.integrator.u, slab_sim.integrator.p)
     return parent(α_land)
 end
 
+"""
+   land_beta(slab_sim::BucketSimulation)
+
+Returns the beta factor of the earth;
+a method for the bucket model
+when used as the land model.
+"""
+function land_beta(slab_sim::BucketSimulation)
+    β_land = ClimaLSM.surface_evaporative_scaling(slab_sim.model, slab_sim.integrator.u, slab_sim.integrator.p)
+    return parent(β_land)
+end
+
 
 """
     get_land_q(slab_sim::Bucketimulation, _...)
 
-Returns the surface specific humidity of the earth; 
-a method for the bucket 
+Returns the surface specific humidity of the earth;
+a method for the bucket
 when used as the land model.
 """
 function get_land_q(slab_sim::BucketSimulation, _...)
```

Comment:
If you would like, you can also use this
https://github.com/CliMA/ClimaLSM.jl/blob/main/src/Bucket/bucket_parameterizations.jl#L46

Reply:
We also have an emissivity function which just returns 1 for the bucket - if that would be useful already to start using

Reply:
This would be great! I think we would need to make it a cached parameter in the Atmos interface, so I'd probably open a new issue for it. :)

Reply:
Would we add a new `land_emissivity` function, or use the bucket emissivity function in an existing coupler function?

Reply:
Just opened a grab bag issue: https://github.com/CliMA/ClimaCoupler.jl/issues/274